common/src/main/java/brocoeur/example/common/blackjack/BlackJackPlay.java

```diff
@@ -4,5 +4,6 @@
 
 public enum BlackJackPlay implements GamePlay {
     STOP,
-    MORE
+    HIT,
+    DOUBLE
 }
```

Comment:
Later on to be added : split -> when receiving two cards the same on the first turn we can split the cards and play a second same bet (so we add the initial amount again) https://youtu.be/gh2FijxPo24

Reply:
yes